File: bigdata-service/src/main/java/com/sofn/bigdata/utils/PropertiesUtil.java
package com.sofn.bigdata.utils;

import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Properties;

public class PropertiesUtil {
    public static Properties getProperties(ClassLoader classLoader, String filePath) throws IOException {
        Properties properties = new Properties();
        InputStreamReader resourceAsStream = new InputStreamReader(classLoader.getResourceAsStream(filePath),"UTF-8");
        properties.load(resourceAsStream);
        return properties;
    }
}

File: bigdata-service/src/main/scala/com/sofn/bigdata/project/utils/PropertiesUtil.scala
package com.sofn.bigdata.project.utils

import java.io.InputStreamReader
import java.util.Properties

object PropertiesUtil {
  def getProperties(classLoader: ClassLoader, path:String, fileName:String)={
    val properties = new Properties();
    val inputStreamReader = new InputStreamReader(classLoader.getResourceAsStream(path+"/"+fileName),"UTF-8")
    properties.load(inputStreamReader)
    properties
  }
}
